activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/tasks/TaskRuntimeUpdateTaskTest.java

```diff
@@ -0,0 +1,153 @@
+package org.activiti.spring.boot.tasks;
+
+import java.util.Date;
+
+import org.activiti.api.runtime.shared.query.Page;
+import org.activiti.api.runtime.shared.query.Pageable;
+import org.activiti.api.runtime.shared.security.SecurityManager;
+import org.activiti.api.task.model.Task;
+import org.activiti.api.task.model.builders.TaskPayloadBuilder;
+import org.activiti.api.task.model.payloads.UpdateTaskPayload;
+import org.activiti.api.task.runtime.TaskAdminRuntime;
+import org.activiti.api.task.runtime.TaskRuntime;
+import org.activiti.spring.boot.RuntimeTestConfiguration;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.MethodSorters;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.security.test.context.support.WithUserDetails;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.assertj.core.api.Assertions.*;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE)
+@ContextConfiguration
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
```

Comment:
We definitely need to find a better way of doing this. I don't like this FixMethodOrder, but for now that's the only way we got it working with `@WithUserDetails` annotation.

Reply:
I don't like this either but this way are configured all tests for tasks due to the user scenarios I believe.

Reply:
yeah, unfortunately we'll need to live with that until we figure out how to switch of user when using `@WithUserDetails`.